src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/CreateCaseTest.java

```diff
@@ -0,0 +1,154 @@
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.controllers.CcdCallbackController;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.dm.DocumentManagementUploadService;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.helper.CaseSearcher;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.helper.EnvelopeMessager;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.CcdApi;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.CcdAuthenticator;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.CcdAuthenticatorFactory;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.events.CreateExceptionRecord;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.logging.appinsights.SyntheticHeaders;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
+import static org.apache.http.HttpHeaders.AUTHORIZATION;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.EventIdValidator.EVENT_ID_CREATE_NEW_CASE;
+
+@SpringBootTest
+@ActiveProfiles("nosb") // no servicebus queue handler registration
+class CreateCaseTest {
+
+    private static final String CASE_REFERENCE = "caseReference";
+
+    @Value("${test-url}")
+    private String testUrl;
+
+    @Autowired
+    private CcdApi ccdApi;
+
+    @Autowired
+    private CaseSearcher caseSearcher;
+
+    @Autowired
+    private EnvelopeMessager envelopeMessager;
+
+    @Autowired
+    private DocumentManagementUploadService dmUploadService;
+
+    @Autowired
+    private CcdAuthenticatorFactory ccdAuthenticatorFactory;
+
+    private String dmUrl;
+
+    @BeforeEach
+    public void setUp() {
+
+        dmUrl = dmUploadService.uploadToDmStore(
+            "Certificate.pdf",
+            "documents/supplementary-evidence.pdf"
+        );
+    }
+
+    @Test
+    public void should_create_case_from_valid_exception_record() throws Exception {
+        // given
+        CaseDetails exceptionRecord = createExceptionRecord("envelopes/new-envelope-create-case-with-evidence.json");
+
+        // when
+        String caseCcdId = invokeCallbackEndpoint(exceptionRecord);
+
+        // then
+        CaseDetails createdCase = ccdApi.getCase(caseCcdId, exceptionRecord.getJurisdiction());
+        assertThat(createdCase.getCaseTypeId()).isEqualTo(exceptionRecord.getCaseTypeId());
+        assertThat(createdCase.getData()).containsExactlyEntriesOf(exceptionRecord.getData());
```

Comment:
I guess these two lines will fail, because the case will have a different case type Id and the data will also be different.

Case type id will be `Bulk_Scanned` and fields in the case will be same as here: https://github.com/hmcts/bulk-scan-ccd-definitions/blob/master/definitions/bulkscan/data/sheets/CaseField.json 

First name, last name and email should be the same as in the exception record's OCR data in the file.